File: translation_output/gpt-3.5/codenet/python/java/p03250.java
```java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int[] abc = Arrays.stream(input).mapToInt(Integer::parseInt).toArray();
        
        Arrays.sort(abc);
        
        int result = Integer.parseInt(String.valueOf(abc[2]) + String.valueOf(abc[1])) + abc[0];
        
        System.out.println(result);
        
        scanner.close();
    }
}
```

File: Results/DER/Translation/translation_output/w_test/granite/codenet/java/python-sanitized/p03250.py
a, b, c = map(int, input().split())

if a >= b and a >= c:
    if b >= c:
        print(a * 100 + b + c)
    else:
        print(a * 100 + c + b)
elif b >= a and b >= c:
    if a >= c:
        print(b * 100 + a + c)
    else:
        print(b * 100 + c + a)
else:
    if a >= b:
        print(c * 100 + a + b)
    else:
        print(c * 100 + b + a)